lib/sql-query-builder/where/geometry-predicate.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function ({ geometry, spatialPredicate = 'ST_Intersects' } = {}) {
+  if (!geometry) return
+  return `${spatialPredicate}(geometry, ?)`
```

Comment:
Can you add a comment to explain the question mark here?

Reply:
Added in 692eb3e.